content/en/developers/ide_integrations/vscode/_index.md

```diff
@@ -39,21 +47,44 @@ You can use the Datadog extension to:
 
 Install the [Datadog Extension][6] from the Visual Studio Marketplace.
 
-## Run Synthetic tests locally
+## Code Insights
```

Comment:
I put the comment here as I can't comment in the section itself but this comment is related to the `Requirements` section. I would probably refactor that a little bit considering that Synthetic tests are not required for the extension to work, and neither having a Datadog Account if we only consider View in VS Code. WDYT?

Reply:
View in VS Code doesn't **strictly** require an account, but the main use-case is clicking links on Datadog and for that you do need an account, so I think this is OK. 

I see your point about the Synthetic Tests - I moved that text to the section about running tests locally.